src/com/visualization/piechart/piesegment.js

```diff
@@ -0,0 +1,34 @@
+import {h, Component} 				from 'preact/preact';
+
+export default class PieSegment extends Component {
+
+	constructor( props ) {
+		super(props);
+
+    }
+
+    render( props ) {
+
+        // it's valid for offset to be zero so we have to check it against undefined.
+        if ( !(props && props['angle'] && props['offset'] != undefined && props['color'])) {
```

Comment:
Prefer a space between the `))`'s at the end (to match the `if ( ` at the front).

Reply:
Also consider my comment for the other file

Reply:
Added spacing